app/changelog/kube-state-metrics.yaml

```diff
@@ -1,3 +1,9 @@
+- version: 1.0.3
+  componentVersion: 1.9.2
+  description: Remove CPU and memory limits.
+  kind: fixed
```

Comment:
```suggestion
  kind: changed
```

Reply:
If for nothing else, then for consistency with equivalent changelog entries done for other apps before.